scope101/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.Trainee">
+        <activity
+            android:name="com.petro.scope101.activity.EnterEmailActivity"
+            android:exported="true" />
+        <activity
+            android:name="com.petro.scope101.activity.EnterPasswordActivity"
+            android:exported="true">
+            <intent-filter>
+                <action android:name="ENTER_PASSWORD" />
```

Comment:
extract to string resources and use @string/xxx to access it.
Also It should be in format com.petro.scope101.ENTER_PASSWORD